aldryn_forms/boilerplates/bootstrap3/templates/aldryn_forms/fields/filefield.html

```diff
@@ -11,7 +11,7 @@
             </label>
         {% endblock form_field_label %}
         {% block form_field %}
-            <input type="file" class="form-control" name="{{ field.name }}" id="{{ field.auto_id }}">
+            <input type="file" class="form-control {{ instance.custom_classes }}" name="{{ field.name }}" id="{{ field.auto_id }}">
```

Comment:
why not do the same here with "|add" to be consistent?

Reply:
not sure i follow. there i had to use |add to concat strings passed as a kwarg to a template tag. here i just render

Reply:
@FinalAngel |add is used in the other places because its a template tag, here is better to not use that.